application/application_db_calls_bench_test.go

```diff
@@ -199,7 +199,7 @@ func (s *BenchDbOperations) BenchmarkGormSelectSpaceRaw() {
 				&sp.Version,
 				&sp.Name,
 				&sp.Description,
-				&sp.OwnerId)
```

Comment:
This error was always there but since this file was ignored by `go build` it was never caught.